### PR TITLE
Changed README to point to github discussion 

### LINKED ISSUE #822: The README points to Spectrum community, which is no longer active
## 🐛 Bug Report

The README points to Spectrum community, which is no longer active

## To Reproduce

Navigate to README and click on the "Community" link

See the Spectrum messages saying "we've moved".

## Expected behavior

Arriving at an active community

## Link to repl or repo (highly encouraged)

https://github.com/gilbarbara/react-joyride#react-joyride

### PR DESCRIPTION
**Pull request by GreenAsJade**
…instead of defunct Spectrum community

Fixes #822.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,14 +8,14 @@
 
 Showcase your app to new users or explain functionality of new features.
 
-It uses [react-floater](https://github.com/gilbarbara/react-floater) for positioning and styling.
```
Comment (gilbarbara) [inline README.md:11]: The double space is required at EOL

File: README.md [hunk 2]
```diff
@@ -8,14 +8,14 @@
...
-It uses [react-floater](https://github.com/gilbarbara/react-floater) for positioning and styling.  
+It uses [react-floater](https://github.com/gilbarbara/react-floater) for positioning and styling.
 And you can use your own components too!
 
 **View the demo [here](https://react-joyride.com/)** (or the codesandbox [examples](https://codesandbox.io/s/github/gilbarbara/react-joyride-demo))
 
 **Read the [docs](https://docs.react-joyride.com/)**
 
-Chat about it in our [Spectrum community](https://spectrum.chat/react-joyride)
+Chat about it in our [Discussion Forum](https://github.com/gilbarbara/react-joyride/discussions)
```
Comment (gilbarbara) [inline README.md:18]: The discussion functionality isn't a "chat" like Spectrum.
I would change it to something like: "Talk about it on the [Discussions board.](...)"

**Review by gilbarbara — CHANGES_REQUESTED** [review]
Minor changes

File: README.md (hunk 1 above)
Comment (GreenAsJade) [reply README.md]: Hah - I am so used to my editor doing that I didn't question it.   Darn Markup mode configed wrong!

**Review by gilbarbara — APPROVED** [review]
LGTM

**Pull request merged**